config/webpack/loaders.js

```diff
@@ -47,12 +47,6 @@ module.exports = [
           minimize: env.NODE_ENV === 'production',
         },
       },
-      {
-        loader: 'postcss-loader',
```

Comment:
So looks like the only reason we're using postcss loader in the first place is that it is a webpacker default (introduced in https://github.com/ManageIQ/manageiq-ui-classic/pull/1132).

So.. yeah we can probably remove it :) .. but if we do that, we should also remove `.postcssrc.yml`.

The plugins we're using are just smart-import (which doesn't help the sass case), precss (css dialect we're not using) and autoprefixer (which may actually be useful).

But if this is hammer/no and we don't have to care about IE, no problem there :).